### PR TITLE
consume official v1beta1 go client in preparation for GA

### PR DESCRIPTION
**Pull request by leiyiz**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: vendor/google.golang.org/api/file/v1beta1/file-gen.go [hunk 1]
```diff
@@ -86,7 +87,30 @@ const (
...
+	if err != nil {
+		return nil, err
+	}
+	if endpoint != "" {
+		s.BasePath = endpoint
+	}
+	return s, nil
+}
+
+// NewServiceBasePath creates a new Service.
```
Comment (saikat-royc) [inline vendor/google.golang.org/api/file/v1beta1/file-gen.go:111]:
```
	svc, err := v1beta1cloudfiler.NewService(ctx, option.WithHTTPClient(httpClient))
	if err != nil {
		return err
	}
	if filestoreEndpoint == AutopushEndpoint {
		svc.BasePath = AutopushBasePath
	} else if filestoreEndpoint == StagingEndpoint {
		svc.BasePath = StagingBasePath
	} else {
		svc.BasePath = ProdBasePath
	}
```
seems to work

Comment (leiyiz) [reply vendor/google.golang.org/api/file/v1beta1/file-gen.go:111]: done

**Pull request merged**